scripts/cmake/CompilerSetup.cmake

```diff
@@ -49,30 +49,38 @@ endif()
 ### GNU C/CXX compiler
 if(COMPILER_IS_GCC)
     if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.9")
-        message(FATAL_ERROR "GCC minimum required version is 4.9! You are using ${CMAKE_CXX_COMPILER_VERSION}.")
+        message(FATAL_ERROR "GCC minimum required version is 4.9! You are \
+            using ${CMAKE_CXX_COMPILER_VERSION}.")
     endif()
     if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug")
         message(STATUS "Set GCC release flags")
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -DNDEBUG")
     else()
         # Enable assertions in STL in debug mode.
-        if (NOT STL_NO_DEBUG)
-            set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -D_GLIBCXX_DEBUG -D_GLIBCXX_DEBUG_ASSERT -D_GLIBCXX_DEBUG_PEDASSERT -D_GLIBCXX_DEBUG_VERIFY")
+        if(NOT STL_NO_DEBUG)
```

Comment:
Ah, I see. Thanks for telling me. But actually we could ask ourselves if that option shouldn't be better removed from our cmake config, because (i) it's rather special, i.e., it is not an OGS feature in any way (ii) every pro-user can set it in their custom cmake config.
But I agree, `STL_NO_DEBUG` is a useful option. If we wanted to remove it, we should put it into some "How to debug" document.